### ISSUE TITLE
How is f2c maintained?

### ISSUE THREAD

**Issue by olebole** [barak/f2c#1]
Hi,

this seems to be a Debian packaging repository for f2c. However, I couldn't find a proper (maintained) upstream.
I've got a few patches that were created by maintaining [IRAF](https://github.com/iraf-community/iraf) where upstream could benefit. Most relevant is https://github.com/iraf-community/iraf/commit/0509cec8c3dc7a636bd20d677ada2026fb1a3ebb. 

Could you give me a hint how to push this to the proper place?

Best

Ole

(also Debian maintainer of IRAF)

**Comment by FranklinChen** [comment]
I found https://netlib.org/f2c/

**Comment by olebole** [comment]
Yes, I know this. It however doesn't look maintained: No contact email is given, and the last modification was on 11 March 2019, while this repository has version 20200916.

**Comment by FranklinChen** [comment]
For what it's worth, https://netlib.org/f2c/changes has an update from 2021:
```
20210928
  README: add a paragraph about the size of integers and a simple
change to f2c.h that is appropriate when sizeof(int) == 4 and
sizeof(long) == 8.
  xsum.c: trivial change to banish a pedantic compiler warning.
```

**Comment by olebole** [comment]
OK, but what is the upstream contact? dmg at acm.org as in the README?

**Comment by barak** [comment]
Yeah, I'm jjust the Debian f2c maintainer. I've been walking the line between tracking upstream (who hates modernity) and a reasonable build system and more up-to-date C code.

Regarding this particular repo, I occasionally download the upstream tarball and import any changes. I try to keep it a mirror of upstream, along with https://salsa.debian.org/debian/libf2c2.git which holds the corresponding f2c library. See `debian/watch` on the debian branch for some code snippets to support that.

There is also https://github.com/barak/f2c-1.git where I put the f2c and libf2c sources into subdirs, and added an autoconf-based build system, and maybe a little more aggressive about tweaking the code.

All around: **_PATCHES WELCOME!!!_**
